### DISCUSSION TITLE
What are the features you would like to see in GenHTTP?

### DISCUSSION THREAD

**Discussion by Kaliumhexacyanoferrat** [Kaliumhexacyanoferrat/GenHTTP#328] [Polls]
It is in the nature of a web server framework that it needs to offer a very wide range of functionality to support the different deployment scenarios well - a web service needs different functionality than a web page or an SPA.

With this survey, I would therefore like to question which topics should be in focus from the community's point of view.

**Comment by amaillo-coop** [comment]
Swagger (OpenAPI)

**Reply by Kaliumhexacyanoferrat** [reply]
@amaillo-coop : Created #436 for this
